### DISCUSSION TITLE
How to use costom embeddings model ?

### DISCUSSION THREAD

**Discussion by sz2three** [towhee-io/towhee#2692] [Q&A]
`    .map('img', 'vec', ops.image_text_embedding.clip(model_name='clip_vit_base_patch32', modality='image'))
`
How to use self defined model in map, instead of `clip` ? 

```
class EmbeddingModel(nn.Module):
    def __init__(self):
        super(EmbeddingModel, self).__init__()
    
    def forward(self, x):
        
        x = self.embedding(x)
         
        return x
```

**Comment by junjiejiangjjj** [comment]
https://towhee.io/image-embedding/timm  Timm op provides many embedding models, you can try it, or follow this doc to use custom operators. https://towhee.readthedocs.io/en/latest/operator/usage.html#custom-operators
